funnel/forms/project.py

```diff
@@ -72,7 +73,14 @@ class ProjectForm(forms.Form):
         ),
         validators=[
             forms.validators.Optional(),
-            forms.validators.ValidUrl(),
+            forms.validators.ValidUrl(
+                allowed_schemes=lambda: current_app.config.get(
+                    'IMAGE_URL_SCHEMES', ('https',)
+                ),
+                allowed_domains=lambda: current_app.config.get('IMAGE_URL_DOMAINS'),
+                message_schemes=__("A https:// URL is required"),
+                message_domains=__("Images must be hosted at images.hasgeek.com"),
```

Comment:
This error is kind of wrong given that the domain is being fetched from config.

Reply:
The expectation is that config will vary between production and development. We're not supporting others with hosting.